server/app.js

```diff
@@ -1,48 +1,91 @@
 const express = require('express');
 const path = require('path');
 const morgan = require('morgan');
-
 const cors = require('cors');
 
 const photoDB = require('../database/index.js');
 
 const app = express();
 
-app.use(cors({
-  origin: '*',
-}));
-
+app.use(express.json());
+app.use(cors({ origin: '*' }));
 app.use(morgan('dev'));
 app.use(express.static(path.join(__dirname, '../public')));
 
-// get all main photos (regular and thumbnail) for all styles by product ID
 app.get('/photos/:productid', async (req, res) => {
   try {
     const productId = Number(req.params.productid);
     const photos = await photoDB.getPhotosByProductId(productId);
-    res.send(photos);
+
+    res.json(photos);
   } catch (err) {
     console.log(err);
-    res.status(400).send({ err: err.message });
+    res.status(500).send('Internal Server Error.');
```

Comment:
Again, sorry for stepping in but to further shorten that line, you could consider res.sendStatus(500) which automatically generates the Internal Server Error message for you! It’s not a crazy change just a QoL change

Reply:
I did not know that. I will keep that in mind. I'll keep it like this for now since it's still 1 line and I'm not sure if I plan on adding any extra stuff to the message.